src/web/pages/tasks/TaskStatus.tsx

```diff
@@ -27,6 +31,7 @@ const TaskStatus = ({task, links = true, isAudit = false}) => {
     report_id = '';
     links = false;
   }
+  const isAudit = task.usageType === USAGE_TYPE.audit;
```

Comment:
nitpick: since it's only used in one place.
```suggestion
 const detailsLinkType =
    task.usageType === USAGE_TYPE.audit ? 'auditreport' : 'report';```